### PR TITLE
Added Python conversation micro tutorial

### PR DESCRIPTION
**Pull request by Victor-Boyd**
I created a micro tutorial section and added the python conversation tutorial to it. If there is anything that should be changed please let me know and I'll get it done! :smile:

### CODE REVIEW

**Review by haixuanTao — COMMENTED** [review]
That's a great example!

**Comment by Victor-Boyd** [comment]
I think I got everything in there correctly this time but let me know if there is anything else I should change! :sweat_smile:

File: docusaurus.config.js [hunk 1]
```diff
@@ -101,6 +102,13 @@ const config = {
             label: "Guides",
             to: "",
           },
+          {
+            type: "docSidebar",
+            sidebarId: "micro_tutorials",
+            position: "left",
+            label: "Micro Tutorials",
+            to: "",
+          },
```
[the remark this answers is not in the document]
Comment (Victor-Boyd) [reply docusaurus.config.js]: Yeah I agree it would probably be a lot cleaner :sweat_smile:

**Review by haixuanTao — APPROVED** [review]
That's awesome! Thanks a lot!

**Pull request merged**